.ebextensions/maintenance.config

```diff
@@ -1,12 +0,0 @@
-files:
-  "/etc/nginx/conf.d/maintenance.conf":
```

Comment:
Do you have another way to have a maintenance page somewhere in here? Could probably live without for awhile.